tephra-core/src/test/java/org/apache/tephra/ThriftTransactionSystemTest.java

```diff
@@ -88,10 +89,14 @@ protected void configure() {
     txClient = injector.getInstance(TransactionSystemClient.class);
     try {
       LOG.info("Starting transaction service");
+      storage.startAndWait();
```

Comment:
It was being stopped without being started, so for consistency I am starting it here. However, start on this object is a no-op.